tests/apiserver-proxy-tests/apiserver-proxy/internal/config.go

```diff
@@ -0,0 +1,80 @@
+package graphql
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/vrischmann/envconfig"
+)
+
+type User string
+
+const (
+	AdminUser    User = "admin"
+	ReadOnlyUser User = "read-only"
+	NoRightsUser User = "no-rights"
+	NoUser       User = "no-user"
```

Comment:
Only `AdminUser` is used, and is passed as unused argument. Remove those constants.